dashboards.go

```diff
@@ -60,516 +42,21 @@ var commandDashboards = cli.Command{
 				cli.StringFlag{Name: "file-path, F", Usage: "read dashboard from the file"},
 			},
 		},
+
+		// urfave/cli will call default Action when running undefined subcommands,
+		// So we leave command defintion to cause error when executing `mkr dashboards generate`.
```

Comment:
I was hoping that mkr dashboards generate would return an error when I deleted the command definition, but it didn't.